index.js

```diff
@@ -34,6 +34,8 @@ function Router (opts) {
   this.tree = new Node()
 }
 
+const forbiddenCharactersInParameterName = ['-']
```

Comment:
This is a bit naive implementation. Rather it should accept regex for each parameter.

Reply:
Can you make (probably again) some examples of regexps that we should support?

How would you implement the check?

Reply:
There is already a complete implementation https://www.npmjs.com/package/path-to-regexp

Reply:
> Rather it should accept regex for each parameter.

Ops, I didn't thought about that. I'll try to extend the current solution in the next days.